features/fixtures/maze_runner/Assets/Scripts/Scenarios/Csharp/Config/MarkLaunchComplete.cs

```diff
@@ -0,0 +1,21 @@
+﻿using BugsnagUnity;
+
+public class MarkLaunchComplete : Scenario
+{
+    public override void PrepareConfig(string apiKey, string host)
+    {
+        base.PrepareConfig(apiKey, host);
+        Configuration.LaunchDurationMillis = 0;
+    }
+
+    public override void Run()
+    {
+        Invoke("DoException", 10);
+    }
+
+    private void DoException()
+    {
+        Bugsnag.MarkLaunchCompleted();
+        Bugsnag.Notify(new System.Exception("MarkLaunchComplete"));
```

Comment:
Even better if you sent one immediately before the `MarkAsCompleted` as well, checking it says isLaunching = true (or do I mean "true"? 😁 ).  Tests are always better if you can demonstrate that taking an action causes something to change.

Reply:
done 👍